src/payment/dto/charge-subscription.dto.ts

```
@@ -1,12 +1,41 @@
 import { ApiProperty } from '@nestjs/swagger';
```

ESLint

[error] Delete `␍`

(prettier/prettier)